src/popup/App.scss

```diff
@@ -1,262 +1,299 @@
-  background-color: #ffff00;
-}
-
-.small-text {
-  font-size: 12px;
-  font-weight: lighter;
-}
-
-.tooltip-inner {
-  max-width: 400px;
-}
-
-.card-rating {
-  border-radius: 8px;
-  border: 1px solid #b02a8f;
-}
-
-/* list aliases */
-.vue-js-switch {
-  margin-bottom: 0;
-}
-
-.list-item-alias .disabled {
-  opacity: 0.7;
-}
-
-.list-item-email {
-  margin-right: 30px !important;
-  position: relative;
-  overflow: hidden;
-  cursor: pointer;
-}
-
-.list-item-email > a {
-  white-space: nowrap;
-}
-
-.list-item-email .email-sub {
-  font-size: 12px;
-}
-
-.list-item-email-fade {
-  right: 0;
-  width: 30px;
-  height: 100%;
-  background: linear-gradient(to right, transparent, white);
-  top: 0;
-  position: absolute;
-}
-
-.list-item-alias .alias-note-preview {
-  font-size: 12px;
-  max-height: calc(12px * 1.5 * 3); /* font-size * line-height */
-  overflow-y: hidden;
-}
-
-.header {
-  .actions-container {
-    position: absolute;
-    right: 0.5rem;
-  }
-  
-  .header-button {
-    height: 20px;
-    margin-top: 2px;
-    margin-left: 10px;
-    margin-right: 7px;
-    color: #b02a8f;
-    cursor: pointer;
-  }
-}
-
-.app-header-menu {
-  left: auto !important;
-  float: right !important;
-  right: 10px;
-  margin-top: 5px !important;
-}
-
-/* toasted: white close button */
-.toasted.toasted-primary > .action.ripple {
-  color: white !important;
-}
-
-/* button img */
-.btn-svg {
-  cursor: pointer;
-  padding: 5px;
-  display: inline;
-}
-
-/* send btn*/
-.btn-send {
-  width: 14px;
-  height: 14px;
-  color: #b02a8f;
-}
-
-/* more options */
-.btn-delete:hover, .btn-svg:hover {
-  background-color: rgba(0, 0, 0, 0.1);
-}
-
-.btn-delete {
-  float: right;
-}
-
-.btn-delete > img {
-  vertical-align: unset;
-  height: 14px;
-}
-
-.more-options {
-  margin-top: 10px;
-}
-
-.cursor {
-  cursor: pointer;
-}
-
-span.link {
-  color: #b02a8f;
-}
-
-.more-options > .action {
-  margin-top: 10px;
-  min-height: 25px;
-}
-
-.more-options > label {
-  margin-bottom: 0;
-  font-size: 12px;
-}
-
-/* BETA badge */
-.beta-badge {
-  padding: 0.1em 0.4em;
-  font-size: 0.65em;
-  margin-left: 1em;
-  border: 0.7px solid #b02a8f;
-  display: inline-block;
-  color: #b02a8f;
-}
-
-/* App Settings */
-table.settings-list > tr {
-  border-bottom: 1px solid #bbb;
-}
-
-table.settings-list > tr > td {
-  vertical-align: top;
-  padding: 6px;
-}
-
-table.settings-list {
-  margin-bottom: 3em;
-
-  tr.disabled {
-    opacity: 0.7;
-    pointer-events: none;
-  }
-}
-
-/* Quick fix for Firefox Overflow Menu */
-.app.ff-overflow-menu {
-  width: auto;
-  font-size: 88%;
-
-  .content {
-    max-height: 500px;
-    overflow-y: auto;
-    scrollbar-width: thin;
-  }
-
-  textarea, input {
-    font-size: 88%;
-  }
-
-  .header {
-    width: 100%;
-
-    .dashboard-btn {
-      height: 20px;
-      margin-top: 2px;
-      padding: 0 0.5rem !important;
-    }
-  }
-}
+@import "./App-color.scss";
+@import "./App-scrollbar.scss";
+@import "./Theme.scss";
```

Comment:
The whole file is marked as changed due to line endings, changed from CRLF to LF.
Let me know if it's significant and you'd like me to revert them.